src/main/kotlin/com/hibob/academy/employeeFeedback/service/FeedbackService.kt

```diff
@@ -78,4 +78,9 @@ class FeedbackService(private val feedbackDao: FeedbackDao) {
 
         } ?: throw NotFoundException("Feedback not found")
     }
+
+    fun changeToReviewedOrUnreviewed(feedbackId: Int, companyId: Int, employeeId: Int, reviewed: Boolean) {
```

Comment:
we don't like 'or' and 'and' in the function names. Usually it's a code smell that the function does more than one thing. Also - if you had more than 2 options, you wouldn't put them all in the function name.
You can name - changeReviewedStatus or something like that

Reply:
Same in the dao